src/gen_proposal.jl

```diff
@@ -0,0 +1,46 @@
+function gen(df::AbstractDataFrame, args...; kwargs...)
+	transform(df, args...; kwargs...)
```

Comment:
first small note (just for the future discussions) - this is not a correct implementation as `args` could overwrite all source columns of `df`.